picard/log.py

```diff
@@ -113,7 +113,7 @@ def formatted_log_line(level, time, message, timefmt='hh:mm:ss',
 
 def _stderr_receiver(level, time, msg):
     try:
-        sys.stderr.write(formatted_log_line(level, time, msg + os.linesep))
+        sys.stderr.write("%r%s" % (formatted_log_line(level, time, msg), os.linesep))
```

Comment:
Yes - I will try it.

However it doesn't mean we won't need the rest of the PR.